addons/mass_mailing/models/mailing.py

```diff
@@ -744,7 +744,9 @@ def _get_seen_list(self):
             SELECT lower(substring(t.%(mail_field)s, '([^ ,;<@]+@[^> ,;]+)'))
               FROM mailing_trace s
               JOIN %(target)s t ON (s.res_id = t.id)
+              %(join_marketing_automation)s
              WHERE substring(t.%(mail_field)s, '([^ ,;<@]+@[^> ,;]+)') IS NOT NULL
+              %(where_marketing_automation)s                               
```

Comment:
weird to have "marketing automation" string in mass mailing, can you rename it `join_domain` or something like that ?